content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: kind
+description: Instructions to setup kind for Istio.
+weight: 21
+skip_seealso: true
+keywords: [platform-setup,kubernetes,kind]
+---
+
+[kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container `nodes`.
+kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+Follow these instructions to prepare kind for Istio installation.
+
+## Prerequisites
+
+- Please use latest Go version, ideally Go 1.13 or greater.
+- To use kind, you will also need to [install docker](https://docs.docker.com/install/).
+- Install the latest version of [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+## Installation steps
+
+1.  Create cluster with the following command:
+
+    {{< text bash >}}
+    $ kind create cluster --name istio-testing
+    {{< /text >}}
+
+    `--name` is used to assign specific name to the cluster. By default, the cluster will be given the name `kind`.
+
+1.  When you want to see the list of clusters, use following command:
+
+    {{< text bash >}}
+    $ kind get clusters
+    kind-istio-testing
```

Comment:
On my Mac, after creating the cluster, the get clusters did not prefix the name:
```
kind get clusters                                                                                                                                                        ✔ istio-testing
```
although it's on the context:
```
CURRENT   NAME                 CLUSTER                        AUTHINFO                                             NAMESPACE
          docker-desktop       docker-desktop                 docker-desktop
          docker-for-desktop   docker-desktop                 docker-desktop
*         kind-istio-testing   kind-istio-testing             kind-istio-testing
          minikube             minikube                       minikube
```

Reply:
Thanks for testing on Mac. I re-created `kind` cluster and I'm also getting only `istio-testing` without `kind` prefix. May be I copied that wrong from `get-context` command. Corrected it.